src/software/ai/hl/stp/play/ball_placement/ball_placement_play.cpp

```diff
@@ -3,8 +3,10 @@
 #include "software/util/generic_factory/generic_factory.h"
 
 
-BallPlacementPlay::BallPlacementPlay(TbotsProto::AiConfig config)
-    : Play(config, true), fsm{BallPlacementPlayFSM{config}}, control_params{}
+BallPlacementPlay::BallPlacementPlay(std::shared_ptr<Strategy> strategy)
+    : Play(true, strategy),
```

Comment:
It may make more sense for `requires_goalie` to be `false` during ball placement. Otherwise, the goalie will still try to goalie